drivers/clock_control/clock_stm32_ll_common.c

```diff
@@ -552,7 +571,11 @@ static int stm32_clock_control_init(const struct device *dev)
 	LL_C2_RCC_SetAHBPrescaler(s_ClkInitStruct->CPU2CLKDivider);
 	LL_RCC_SetAHB4Prescaler(s_ClkInitStruct->AHB4CLKDivider);
 #endif /* CONFIG_SOC_SERIES_STM32WBX */
-
+#ifdef CONFIG_SOC_SERIES_STM32WLX
+	/* Set C2 AHB & AHB4 prescalers */
+	LL_C2_RCC_SetAHBPrescaler(s_ClkInitStruct->CPU2CLKDivider);
+	LL_RCC_SetAHB3Prescaler(s_ClkInitStruct->AHB3CLKDivider);
```

Comment:
comment says "Set C2 AHB & AHB**4** prescalers"  and then LL_RCC_SetAHB**3**Prescaler()

Reply:
Fixed